src/test/scala/com/lightbend/kafka/scala/streams/typesafe/StreamToTableJoinScalaIntegrationTestMixImplicitSerialized.scala

```diff
@@ -0,0 +1,161 @@
+/**
+  * Copyright (C) 2018 OpenShine SL <https://www.openshine.com>
```

Comment:
I don't think the copyright attributions are necessary, unless the intention is to release a fork of this library by OpenShine, in which case they should live there.